Server/Utilities/SpringBoardAlerts.m

```diff
@@ -48,25 +32,36 @@ - (instancetype)init {
 - (instancetype)init_private {
     self = [super init];
     if (self) {
-        _alerts = [self concatenateArrays:@[
-                                            [self USEnglishAlerts],
-                                            [self DanishAlerts],
-                                            [self DutchAlerts],
-                                            [self DutchAlerts_BE],
-                                            [self SwedishAlerts],
-                                            [self EUSpanishAlerts],
-                                            [self ES419SpanishAlerts],
-                                            [self GermanAlerts],
-                                            [self FrenchAlerts],
-                                            [self RussianAlerts],
-                                            [self PTBrazilAlerts],
-                                            [self KoreanAlerts],
-                                            [self ItalianAlerts],
-                                            [self HebrewAlerts],
-                                            [self GreekAlerts],
-                                            [self HungarianAlerts],
-                                            [self CzechAlerts]
-                                            ]];
+        NSDataAsset *asset = [[NSDataAsset alloc] initWithName:@"springboard-alerts" bundle: [NSBundle bundleForClass:[self class]]];
```

Comment:
Wrap lines to ~80 chars.

Use Xcode to do the formatting.

There is a style guide: https://github.com/calabash/DeviceAgent.iOS/blob/develop/STYLE_GUIDE.md

Reply:
Fixed